src/main/kotlin/com/craftschemers/hub/managers/HubCommandManager.kt

```diff
@@ -13,6 +14,8 @@ class HubCommandManager : CommandExecutor {
     private val commands = mapOf(
         JoinCommand.name to JoinCommand,
         LeaveCommand.name to LeaveCommand,
+        SetCommand.name to SetCommand,
+        PrintPlayersInGameCommand.name to PrintPlayersInGameCommand
```

Comment:
Trailing comma please
```suggestion
        PrintPlayersInGameCommand.name to PrintPlayersInGameCommand,
```